integrations/opensearch-document-store.md

```diff
@@ -28,6 +28,7 @@ toc: true
 - [Overview](#overview)
 - [Installation](#installation)
 - [Usage](#usage)
+- [Hybrid Retriever](#hybrid-retriever)
```

Comment:
I would leave it out here. It's just one of multiple usage examples and we're also not listing `Writing Documents to OpenSearchDocumentStore`, which is on the same level as `Hybrid Retriever`
```suggestion
```